Tests/AsyncAwait/ChatAsyncIntegrationTests.swift

```diff
@@ -221,6 +260,30 @@ class ChatAsyncIntegrationTests: BaseAsyncIntegrationTestCase {
     }
   }
 
+  func testChatAsync_GetChannelsWithPagination() async throws {
+    let channel1 = try await chat.createChannel(id: randomString(), name: "Channel1")
```

Comment:
Shouldn't channel ids have prefix "swift-chat"?

Reply:
The `randomString()` has an optional `withPrefix:` argument (default: true) that prepends `swift-chat`. There's no need to pass it explicitly unless overriding the default. I will extract `swift-chat` as a constant to not to repeat it in many places